mappings/src/androidMain/kotlin/com/revenuecat/purchases/kmp/mappings/Offering.android.kt

```diff
@@ -0,0 +1,36 @@
+package com.revenuecat.purchases.kmp.mappings
+
+import com.revenuecat.purchases.kmp.Offering
+import com.revenuecat.purchases.kmp.Package
+import com.revenuecat.purchases.Offering as NativeAndroidOffering
+
+public fun NativeAndroidOffering.toOffering(): Offering =
+    AndroidOffering(this)
+
+public fun Offering.toAndroidOffering(): NativeAndroidOffering =
```

Comment:
We need to make `Offering` an interface and hold the original object since we need to pass it in for paywalls, and it has the paywall data inside.